test/bigtest/tests/title-show-test.js

```diff
@@ -56,6 +56,10 @@ describe('TitleShow', () => {
       expect(TitleShowPage.nameHasFocus).to.be.true;
     });
 
+    it('does not display the edit button for a managed title', () => {
```

Comment:
We probably want `it.always` here unless the button was previously visible

Reply:
Added.